src/interfaces/IBPool.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.23;
+
+interface IBPool {
+  struct Record {
+    bool bound; // is token bound to pool
+    uint256 index; // internal
+    uint256 denorm; // denormalized weight
+  }
+
+  event LOG_SWAP(
+    address indexed caller,
+    address indexed tokenIn,
+    address indexed tokenOut,
+    uint256 tokenAmountIn,
+    uint256 tokenAmountOut
+  );
+
+  event LOG_JOIN(address indexed _caller, address indexed _tokenIn, uint256 _tokenAmountIn);
```

Comment:
I think we should keep it, but tbh I dont remember the wonder standard

Reply:
i'd leave them w/o underscores since is Balancer standard apparently, for events it'll break their SDK if we change it, and for interfaces is no different, and they don't use underscored arguments